src/main/java/uk/gov/digital/ho/hocs/audit/export/ExportDataConverter.java

```diff
@@ -31,15 +34,32 @@ public ExportDataConverter(InfoClient infoClient, CaseworkClient caseworkClient)
         }
 
         for (int i = 0; i < auditData.length; i++){
-            if (uuidToName.containsKey(auditData[i])){
-                auditData[i] = uuidToName.get(auditData[i]);
+            String uuidData = auditData[i];
+            if (!isUUID(uuidData)) {
+                continue;
+            }
+            if (uuidToName.containsKey(uuidData)){
+                auditData[i] = uuidToName.get(uuidData);
+            } else {
+                GetCaseReferenceResponse caseReferenceResponse = caseworkClient.getCaseReference(uuidData);
+                if (caseReferenceResponse.getReference() != null && !caseReferenceResponse.getReference().isEmpty()) {
```

Comment:
Use 'StringUtils.hasValue(caseReferenceResponse.getReference())' for better code readability here.

Reply:
Done but still showing in diff.